project-templates/ecs/basic/README.md

```diff
@@ -0,0 +1,161 @@
+1. Inside Terraform Cloud, Click Registry and Select `Publish a Module`
+![img.png](../readme-images/tfc_publish_module_ss.png)
+
+
+2. Select Github under `Version Control Provider`
+
+![img_1.png](../readme-images/publish_module_vcs_ss.png)
+
+3. Select your forked repository of the no-code module created in the last step.
+
+![img_2.png](../readme-images/select_repo_ss.png)
+
+4. Click `Publish Module`
+
+![img_3.png](../readme-images/publish_module.png)
+
+5. Review the published module details
+
+![img_4.png](../readme-images/review_module_details.png)
+
+
+
+![img_8.png](../readme-images/tfc_registry_ss.png)
+
+### Terraform Cloud Workspaces
+
+1. Create terraform cloud [workspaces](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/creating) in the TFC User Interface based on the following terraform configurations provided in this example:
+        `terraform-aws-example-microservice-infra`
+        `terraform-aws-example-network`
+![img_9.png](../readme-images/create_tfc_workspace_ss.png)
+
+
+2. Go to Settings for the created TFC workspaces, Under `Remote State Sharing` allow `Share with all workspaces in this organization`.
+![img_10.png](../readme-images/tfc_remote_sharing_ss.png)
+
+
+3. Create a [global variable set](https://developer.hashicorp.com/terraform/tutorials/cloud-get-started/cloud-workspace-create#create-the-workspace) for your AWS Credentials.
+
+### AWS Steps with Terraform
+
+Create the baseline infrastructure with the following steps in your CLI.
+
+1. Navigate to the `terraform-aws-example-network` directory found in this example and run `terraform init` and the `terraform apply` command.
+
+2. Navigate to the `terraform-aws-example-microservice-infra` directory found in this example and run `terraform init` and the  `terraform apply` command.
+
+
+## Waypoint Template Steps 
+
+TODO(Teresa): Replace figma images with proper screenshots once UI is ready.
+
+
+1. Set up your TFC Credentials in Waypoint. You will need a TFC team access token found in Terraform Cloud and your TFC Organization Name.
+![img.png](../readme-images/waypoint_tfc_creds_setup_ss.png)
+
+
+2. Create a Waypoint Project Template by filling in the fields in the Template Creation page:
+![img_1.png](../readme-images/waypoint_create_template_ss.png)
+
+
+
+3. Return to your Projects List Page. In the top right corner, select `Create Project`. From the dropdown menu, select `Create Project with template`:
+![img_3.png](../readme-images/waypoint_projects_list_ss.png)
+
+   
+4. Select the template you would like to use for this project.
+![img_4.png](../readme-images/waypoint_select_template_ss.png)
+ 
+   
+
+5. Fill in the name of your project under `Project Name`. Then click `Create Project.
+![img_6.png](../readme-images/waypoint_create_project_ss.png)
+
+   
+6. Review the project details of your newly created project! You're Ready to Go!
+![img_7.png](../readme-images/waypoint_project_details_ss.png)
+
```

Comment:
A few more things I think we need here:

- Mention that clicking the create button triggers a terraform cloud workspace run, which will take some time. Point out the spinner pointing to the workspace.
- Run through connecting a repo and deploying and releasing successfully. I think the easiest thing is probably to copy the [ecs nodejs example](https://github.com/hashicorp/waypoint-examples/tree/main/aws/aws-ecs/nodejs) into this folder, and remove the `waypoint.hcl` file. 
- A conclusion!